client/extensions/woocommerce/app/order/order-details/index.js

```diff
@@ -43,10 +43,10 @@ class OrderDetails extends Component {
 		}
 	};
 
-	updateLineItems = item => {
+	updateOrder = newOrder => {
```

Comment:
Must figure out way to have argument named `depecheMode` sometime

Reply:
Ha - almost left a comment with a link to _blue monday_ when I read this too.